p2p/libp2p/netMessenger.go

```diff
@@ -152,7 +153,7 @@ func newNetworkMessenger(args ArgsNetworkMessenger, messageSigning messageSignin
 		return nil, fmt.Errorf("%w when creating a new network messenger", p2p.ErrNilPeersRatingHandler)
 	}
 
-	p2pPrivKey, err := createP2PPrivKey(args.P2pConfig.Node.Seed)
+	p2pPrivKey, err := createP2PPrivKey(args.P2pPrivKeyBytes)
```

Comment:
where is the cleanup for the p2p/libp2p/rand package? We should delete that package